### PR TITLE
[zh-cn]fix-some-overdue-in-debug-cluster/windows.md

### PR DESCRIPTION
**Pull request by yanrongshi**
https://kubernetes.io/zh-cn/docs/tasks/debug/debug-cluster/windows/
image
image

### CODE REVIEW

File: content/zh-cn/docs/tasks/debug/debug-cluster/windows.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ content_type: concept
 1. 我的 Pod 都卡在 “Container Creating” 或者不断重启
 
    确保你的 pause 镜像跟你的 Windows 版本兼容。
-   查看 [Pause 容器](/zh-cn/docs/setup/production-environment/windows/intro-windows-in-kubernetes#pause-container)
+   查看 [Pause 容器](/zh-cn/docs/concepts/windows/intro/#pause-container)
    以了解最新的或建议的 pause 镜像，或者了解更多信息。
```
Comment (Sea-n) [inline content/zh-cn/docs/tasks/debug/debug-cluster/windows.md:35]: The commented original text should be updated too.

Comment (yanrongshi) [reply content/zh-cn/docs/tasks/debug/debug-cluster/windows.md:35]: Thank you very much for your reminding. I will revise it

Comment (yanrongshi) [reply content/zh-cn/docs/tasks/debug/debug-cluster/windows.md:35]: @Sea-n cc

**Pull request merged**